lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -84,6 +84,13 @@ def main(argv=None):
     parser.add_argument(
         'output_filepath', metavar='OUTPUT_FILEPATH',
         help='The output path for the processed NetCDF')
+    parser.add_argument(
+        'distribution', metavar="DISTRIBUTION",
+        help="The distribution for constructing realizations, percentiles or "
+             "probabilities. This should typically match the distribution "
+             "used for minimising the Continuous Ranked Probability Score "
+             "when estimating the EMOS coefficients. The distributions "
+             "available are those supported by scipy.stats.")
```

Comment:
```Currently tested distributions are: norm, truncnorm``` or something along those lines? It would be nice to provide an example distribution name in this way so people can play with this without having to go and lookup scipy.stats; it also indicates the limits of our experience with the different distributions.

Reply:
I've added this sentence as suggested, so that it's clearer regarding which distributions have been tested.